src/libexpr/primops/fetchTree.cc

```diff
@@ -39,6 +40,10 @@ void emitTreeAttrs(
         attrs.alloc("submodules").mkBool(
             fetchers::maybeGetBoolAttr(input.attrs, "submodules").value_or(false));
 
+    if (input.getType() == "git")
+        attrs.alloc("exportIgnore").mkBool(
+            fetchers::maybeGetBoolAttr(input.attrs, "exportIgnore").value_or(false));
+
```

Comment:
I think we can leave this out. It's not really useful to return attributes that the caller already knows (unlike e.g. `rev`).  (Yeah the same applies to `submodules`, but that's too late...)

Reply:
Reminds me of derivation/derivationStrict. Agreed and done.